Bikeini/screens/AddBike.js

```diff
@@ -0,0 +1,493 @@
+          },
+        ],
+        tandem: [
+          {
+            label: 'Single',
+            value: 2,
+          },
+          {
+            label: 'Tandem',
+            value: 1,
+          },
+        ],
+        rack: [
+          {
+            label: 'Rack',
+            value: 1,
+          },
+          {
+            label: 'No Rack',
+            value: 2,
+          },
+        ],
+        basket: [
+          {
+            label: 'Basket',
+            value: 1,
+          },
+          {
+            label: 'No Basket',
+            value: 2,
+          },
+        ],
+        mudguard: [
+          {
+            label: 'Mudguard',
+            value: 1,
+          },
+          {
+            label: 'No Mudguard',
+            value: 2,
+          },
+        ],
+        chainProtection: [
+          {
+            label: 'Chain protector',
+            value: 1,
+          },
+          {
+            label: 'No chain protector',
+            value: 2,
+          },
+        ],
+        net: [
+          {
+            label: 'Net',
+            value: 1,
+          },
+          {
+            label: 'No Net',
+            value: 2,
+          },
+        ],
+        winterTires: [
+          {
+            label: 'Winter Tires',
+            value: 1,
+          },
+          {
+            label: 'Summer Tires',
+            value: 2,
+          },
+        ],
+        light: [
+          {
+            label: 'Light',
+            value: 1,
+          },
+          {
+            label: 'No Light',
+            value: 2,
+          },
+        ],
+      },
+      Color: [
+        {
+          value: 'Red',
+        },
+        {
+          value: 'Not Red',
+        },
+      ],
+    };
+    this.cameraRollPermission = permissions.cameraRollPermission.bind(this);
+  }
+
+  startCameraRoll = () => {
+    this.cameraRollPermission(this.pickImage);
+  }
+
+  pickImage = async () => {
+    const { hasCameraRollPermission } = this.state;
+    if (!hasCameraRollPermission) {
+      Alert.alert('No Access');
+      return;
+    }
+    const result = await ImagePicker.launchImageLibraryAsync({
+      allowsEditing: true,
+      aspect: [4, 3],
+    });
+
+    if (!result.cancelled) {
+      const { saveImageToState } = this.props;
+      saveImageToState(result.uri);
+    }
+  };
+
+  sendBikeToServer = () => {
+    const { addBikeState, authState } = this.props;
+    const { bikeData } = this.state;
+    const bikeImgData = {
+      uri: addBikeState.uriSet,
+      name: `${bikeData.addType}.jpg`,
+      type: 'image/jpg',
+    };
+    this.setBikeData('imageOfBike', bikeImgData);
+    serverApi.fetchApi('bikes/addbike', bikeData, 'multipart/form-data', authState.jwt[0]);
+  }
+
+  setBikeData = (attr, value, setKeyword) => {
+    const { bikeData } = this.state;
+    if (setKeyword) {
+      const { keywords } = bikeData;
+      if (value === 1 || value === 2) {
+        keywords[attr] = Boolean(value);
```

Comment:
Re-assigning values of const-varibles. Not sure if it gives error? (Also applies to line 274 and 277 if so)

Reply:
according to ESlint this is correct procedure. Not certain why since I change stuff but I get an error if I do "let". Might be rules I don't know about regarding changing sub attributes